tests/test_post_view.py

```diff
@@ -0,0 +1,39 @@
+from audioop import reverse
+from urllib import response
+from django.test import TestCase ,Client
+from django.urls import reverse_lazy
+from post.models import Post
+
+class HomePageTest(TestCase):
+
+
+      fixtures = [
+      "user_test.json",
+      "category_test.json",
+      "post_test.json",
+      "comment_test.json",
+        ]
+        
+      def setUp(self):
+        self.client=Client()
+        self.home_url = reverse_lazy
+        
+        
+      def test_home(self):
+          '''test homepage, post home view'''
+          self.client.login(username="anna", 
+          password="password")
+          
+          response=self.client.get(self.home_url)
```

Comment:
You are passing here just a reference to reverse_lazy without an argument (name of url path), this will throw an error:)

Reply:
yes , you are right , I rewrote it , and commit in the latest commit;  I would like to just delete this one, and merge test_post_model.py  if that one is ready. :)